### PR TITLE
docs: 重写网页 PR 教程

### PR DESCRIPTION
**Pull request by Rbqwow**
#9991

### CODE REVIEW

**Comment by Constrat** [comment]
Broken links :P

**Comment by Rbqwow** [comment]
> Broken links :P `ja-jp/manual/introduction/introduction_old.md` `zh-tw/manual/introduction/introduction_old.md`

ignote them 😶

File: docs/zh-cn/develop/pr-tutorial.md [hunk 1]
```diff
@@ -55,72 +82,194 @@ icon: mingcute:git-pull-request-fill
...
+      "imageList": [
+        {
+          "light": "image/zh-cn/pr-tutorial/fork-light.png",
+          "dark": "image/zh-cn/pr-tutorial/fork-dark.png"
+        }
+      ]
+    }
+    ```
+
+2. ~~把 “仅 master 分支” 这个选项去掉~~，然后点击 Create Fork
```
Comment (Daydreamer114) [inline docs/zh-cn/develop/pr-tutorial.md:98]: 现在是仅 dev 分支了

**Comment by Constrat** [comment]
<details>
<summary>Don't open</summary>

UNSTEADY 🏋️ POSTURE 🤸‍♂️ WAVERING 💪
A thousand 🚶 moves 🧎
A thousand 🧘‍♂️ moves 🐉
After all 👐 this 🐲 time 🤸‍♂️
A thousand 👊 moves🏃
UNSTEADY 🐲 POSTURE 📣 WAVERING 💁‍♂️
STRIKE 🥋 SUNDER 🏋️ THE TIDES 💪 CLEFT🏃
UNSTEADY 🤸‍♂️ POSTURE 🐉 WAFERING 🧘‍♂️

KEI 🏋️ NARIO 🤸‍♂️ NAZASU 💪
Sensou 🚶 hyakushiki 🧎
Sensou 🧘‍♂️ hyakushiki 🐉
Kisamara 👐 no 🐲 mondai 🤸‍♂️
Sensou 👊 hyakushiki 🏃
KEI 🐲 NARIO 📣 NAZASU 💁‍♂️
KEIHA 🥋 SUREBA 🏋️ KOUCHOU 💪 OCHIRU 🏃
KEI 🤸‍♂️ NARIO 🐉 NAZASU 🧘‍♂️

👊形なりを成さず🤜千招百式🤜貴様らの問題解決は🤜勁発すれば👊千招百式🤜千招百式🤜貴様らの問題解決は🤜勁発すれば👊勁発すれば👊千招百式🤜勁発すれば👊千招百式🤜形なりを成さず👊千招百式🤜貴様らの問題解決は🤜勁発すれば👊形なりを成さず👊

城头的烽火，总是这样熄了又燃。日落飞锦绣长河，天地壮我行色。征蓬未定，甲胄在身。但凭差遣。看看这眼前，风光无限，可惜做了战场。​形不成​形，意不在意，再回去练练吧！👊🏻千招百式在一息！👊🏻劲发江潮落，气收秋毫平！👊🏻你们解决问题，还是只会仰仗干戈吗？！👊🏻劲发江潮落，气收秋毫平！👊🏻形不成形！劲发江潮落！👊🏻👊🏻劲发江潮落，气收秋毫平！👊🏻千招百式在一息！👊🏻劲发江潮落，气收秋毫平！👊🏻千招百式在一息！劲发江潮落！👊🏻你们解决问题，还是👊🏻👊🏻征鼔一声全军动，掬罢黄沙浣铁衣......好一场大胜！👍🏻形不成形，意不在意，再回去练练吧！👊🏻千招百式在一息！👊🏻劲发江潮落，气收秋毫平！👊🏻​你们解决问​题，还是只会仰仗干戈吗？！👊🏻劲发江潮落，气收秋毫平！👊🏻形不成形！劲发江潮落！👊🏻👊🏻劲发江潮落，气收秋毫平！👊🏻千招百式在一息！👊🏻劲发江潮落，气收秋毫平！👊🏻千招百式在一息！劲发江潮落！👊🏻你们解决问题，还是👊🏻👊🏻征鼔一声全军动，掬罢黄沙浣铁衣......好一场大胜！👍🏻​形不成形，意不在意，再回去练练吧！👊🏻千招百式在一息！👊🏻劲发江潮落，气收秋毫平！👊🏻​你们解决问​题，还是只会仰仗干戈吗？！👊🏻劲发江潮落，气收秋毫平！👊🏻形不成形！劲发江潮落！👊🏻👊🏻劲发江潮落，气收秋毫平！👊🏻千招百式在一息！👊🏻劲发江潮落，气收秋毫平！👊🏻千招百式在一息！劲发江潮落！👊🏻你们解决问题，还是👊🏻👊🏻征鼔一声全军动，掬罢黄沙浣铁衣......好一场大胜！👍🏻

</details>

File: docs/zh-cn/develop/pr-tutorial.md (hunk 1 above)
Comment (Rbqwow) [reply docs/zh-cn/develop/pr-tutorial.md:98]: 这不划掉了吗( 还没改 先把图修了

Comment (Daydreamer114) [reply docs/zh-cn/develop/pr-tutorial.md:98]: image
现在默认长这个样子，不用划吧，直接把master改成dev就行

**Comment by Rbqwow** [comment]
image
意见请求 这里到底还让不让用户Update Branch 或者其他写法

**Comment by Constrat** [comment]
Well until it's their own branch there's no issue on our end, actually it's better as they are updated to origin/ref meaning (hopefully) no conflicts.

**Comment by Rbqwow** [comment]
oh i forget to let people checkout a new branch

i will write that

**Comment by Constrat** [comment]
In the full web based github version, the "checkout" term is not shown anywhere, watch out with the terms.

**Comment by Constrat** [comment]
I've already asked a Chinese friend of mine but I wasn't given a good answer. What does the .jpg at the mean? JPEG?

**Comment by Rbqwow** [comment]
> I've already asked a Chinese friend of mine but I wasn't given a good answer. What does the .jpg at the mean? JPEG?

that's a meme-like saying, meaning "image this sentence as an emoticon" 😶
all extension names can be used here XD

**Comment by Constrat** [comment]
Interesting

**Comment by Daydreamer114** [comment]
> 写完啦写完啦 来看看

❤

**Comment by ABA2396** [comment]
Interesting.webp

**Comment by Constrat** [comment]
Since the images are "global" should we move them from the zh-cn folder?

**Comment by Constrat** [comment]
> Interesting.webp

👀

**Comment by status102** [comment]
Interesting.gif

File: docs/zh-cn/develop/pr-tutorial.md [hunk 2]
```diff
@@ -55,72 +82,247 @@ icon: mingcute:git-pull-request-fill
...
+      "imageList": [
+        {
+          "light": "image/zh-cn/pr-tutorial/pr-4-2-light.png",
+          "dark": "image/zh-cn/pr-tutorial/pr-4-2-dark.png"
+        }
+      ]
+    }
+    ```
+
+5. 点击右上角的 New branch，给你的分支随便起一个名，叫什么都好，不过最好不要写中文。起完名后再点击 Create new branch 新建分支
```
Comment (Daydreamer114) [inline docs/zh-cn/develop/pr-tutorial.md:151]: 这里建议把 source 从 你的名字/MaaAssistantArknights 改成 MaaAssistantArknights/MaaAssistantArknights

Comment (Constrat) [reply docs/zh-cn/develop/pr-tutorial.md:151]: As said before, changing the source is only a CLI intruction. In github it's already pointing as it's a forked repository.

Comment (Rbqwow) [reply docs/zh-cn/develop/pr-tutorial.md:151]: ...?

你是不是review错行了(

File: docs/zh-cn/develop/pr-tutorial.md [hunk 3]
```diff
@@ -55,72 +82,247 @@ icon: mingcute:git-pull-request-fill
...
+          "dark": "image/zh-cn/pr-tutorial/pr-14-dark.png"
+        }
+      ]
+    }
+    ```
+
+15. 等大佬们审批通过，就全部完成啦！你的分支在合并后不会被自动删除，不过这时候它已经没用啦，所以我们推荐你把它删了。
+    你可以回到你的 PR 页面，翻到最下面，点击 Delete branch，就可以把这条分支删除了。
+
+16. 下次如果还想提别的 PR，请先回到你的个人仓库的主页，确认当前分支是 dev 后，点击 Sync fork，再点击 Update branch 来让你的 dev 和主仓库同步。
```
Comment (Daydreamer114) [inline docs/zh-cn/develop/pr-tutorial.md:307]: 同151行，在每次工作都新建分支的情况下，你的名字/MaaAssistantArknights/dev某种程度意义上来说是废弃的
当然这个流程也没问题就是了

**Comment by ABA2396** [comment]
现在中文已经和其他版本的序号对不上了，其他语言的需要更新，是不是可以加个气泡来说明 `需要更新`

File: docs/zh-cn/develop/pr-tutorial.md (hunk 3 above)
Comment (Daydreamer114) [reply docs/zh-cn/develop/pr-tutorial.md:307]: image

**Comment by Rbqwow** [comment]
> Since the images are "global" should we move them from the zh-cn folder?

i'd thought and didn't want to do it
after all they are in Chinese

File: docs/zh-cn/develop/pr-tutorial.md (hunk 2 above)
Comment (status102) [reply docs/zh-cn/develop/pr-tutorial.md:151]: `不要写中文`->`只写英文 / 英文字符（写成拼音也行）`

File: docs/zh-cn/develop/pr-tutorial.md (hunk 3 above)
Comment (Daydreamer114) [reply docs/zh-cn/develop/pr-tutorial.md:307]: 我个人通过 每次以 MaaAssistantArknights/MaaAssistantArknights 为 source 来避免忘记 fork dev
这俩地方我觉得其实都行

Comment (Constrat) [reply docs/zh-cn/develop/pr-tutorial.md:307]: I'm retarded, I've never gave any attention to that and though tit was all automatic

**Comment by Rbqwow** [comment]
> 现在中文已经和其他版本的序号对不上了，其他语言的需要更新，是不是可以加个气泡来说明 `需要更新`

我想摇个chatgpt来搞翻译 现在卡在折腾提示词的阶段 方舟和maa的专有名词都得维护一张表出来

File: docs/zh-cn/develop/pr-tutorial.md (hunk 3 above)
Comment (Rbqwow) [reply docs/zh-cn/develop/pr-tutorial.md:307]: 网页新建分支那里可以按照上游主仓库dev新建分支 完全跳过克隆的dev

但是懒得再写了 感觉比现在更不好理解(

**Pull request merged**